common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -85,6 +85,7 @@ remove_racial_traits_effect = {
 	if = { limit = { trait = creature_black_wyrm 			} remove_trait = creature_black_wyrm 			}
```

Comment:
Shoudn't gilblins be added here?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/109885665-59e53980-7c98-11eb-91cc-b033c33f911c.png)

</details>